MapleServer2/PacketHandlers/Game/RequestUserEnvHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class RequestUserEnvHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.REQUEST_USER_ENV;
+
+        public RequestUserEnvHandler(ILogger<RequestUserEnvHandler> logger) : base(logger) { }
+
+        private enum TitleMode : byte
+        {
+            Unk = 0x0,
+            Change = 0x1,
+            Trophies = 0x3,
+        }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            TitleMode mode = (TitleMode) packet.ReadByte();
+
+            switch (mode)
+            {
+                case TitleMode.Unk:
+                    break;
+                case TitleMode.Change:
+                    HandleTitleChange(session, packet);
+                    break;
+                case TitleMode.Trophies:
```

Comment:
Do we know that this is for trophies? And also if we know that this packet is used for more than just titles, let's change the `TitleMode` to `UserEnvMode`

Reply:
I think it's related to the Trophies UI to load what trophies the player has